package.json

```diff
@@ -43,10 +43,11 @@
   "devDependencies": {
     "@faker-js/faker": "^8.4.1",
     "@types/jest": "^26.0.15",
+    "@types/node": "^12.11.5",
+    "dotenv": "^16.4.5",
```

Comment:
Fine adding that this way for now. I've created [PR#69](https://github.com/onfido/onfido-openapi-spec/pull/69) to not forget about this change.